superbench/benchmarks/micro_benchmarks/cublas_function/cmd_helper.h

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//Utility for storing and parsing command line arguments
+
+#pragma once
+
+#include <iostream>
```

Comment:
please run `make cpplint `to check the code style and `make cppformat` to auto format c++ code.